### ISSUE TITLE
Automatic linking with marked functions

### ISSUE THREAD

**Issue by sornas** [FredTheDino/sylt-lang#27]
For example:

```rust
#[tihdy_extern]
pub fn callable_from_game(x: f32, y: f32) {
    code
}
```

**Comment by FredTheDino** [comment]
It turns out, this is impossible with how the compiler is set up. There's no way to order the execution of macros. This would require all methods to be positioned before the list we want to create, and we still need to give in the path for each function.

And this is why noone else does it in Rust.

**Comment by sornas** [comment]
Closed with 5449818b0975341e03720a450eec877453dee7c4 via #80
